spec.md

```diff
@@ -371,26 +362,12 @@ authenticated before use.
 ## Policy Validation
 
 When sending to an MX at a domain for which the sender has a valid and
-non-expired SMTP STS policy, a sending MTA honoring SMTP STS SHOULD validate
+non-expired SMTP STS policy, a sending MTA honoring SMTP STS MUST validate
```

Comment:
Since we're planning to move reporting to its own thing, I don't feel it's
necessary here.

/m

On Wed, Apr 6, 2016 at 12:40 PM danmarg <notifications@github.com> wrote:

> In spec.md
> <https://github.com/mrisher/smtp-sts/pull/21#discussion_r58728391>:
>
> > @@ -371,26 +362,12 @@ authenticated before use.
> >  ## Policy Validation
> >
> >  When sending to an MX at a domain for which the sender has a valid and
> > -non-expired SMTP STS policy, a sending MTA honoring SMTP STS SHOULD validate
> > +non-expired SMTP STS policy, a sending MTA honoring SMTP STS MUST validate
>
> Do we want to call out the possibility for a sending MTA to be
> non-STS-aware but to still do offline reporting? Or is that not necessary?
>
> —
> You are receiving this because you authored the thread.
>
> Reply to this email directly or view it on GitHub
> <https://github.com/mrisher/smtp-sts/pull/21/files/255bb14be5dfefe320c772dc15947c4b3b6ae071#r58728391>
>